src/core/CMakeLists.txt

```diff
@@ -152,15 +152,15 @@ ENDIF (ANDROID)
 
 FIND_PACKAGE(Sqlite3)
 
-INCLUDE_DIRECTORIES(SYSTEM
+TARGET_INCLUDE_DIRECTORIES(qgis_core SYSTEM INTERFACE 
   ${CMAKE_SOURCE_DIR}/src/core/qgsquick
   ${QGIS_INCLUDE_DIR}
   ${PROJ_INCLUDE_DIR}
   ${GDAL_INCLUDE_DIR}
   ${SQLITE3_INCLUDE_DIR}
 )
 
-INCLUDE_DIRECTORIES(
```

Comment:
I think they are public (or interface? I don't understand the implications completely) as well.

Reply:
I was not sure, but that makes sense yes.

Reply:
then I'd remove them from the other CMakeFiles (app, tests, ...) in the same PR since they are now inherited.

Reply:
done